css-brochure-form/styles.css

```diff
@@ -0,0 +1,73 @@
+h1,
+label,
+button {
+  font-family: 'Brandon Grotesque', sans-serif;
+  text-transform: uppercase;
+  font-weight: 600;
+}
+
+h1 {
+  font-size: 15px;
+  color: rgb(236, 93, 87);
+  text-align: center;
+  margin: 14px auto 10px;
+  padding: 0;
+}
+
+form {
+  width: 383px;
+  margin: 0 auto;
+  padding: 0;
+}
+
+div {
+  height: 73px;
+}
```

Comment:
Rather than setting a `height` to your elements for spacing, I would recommend using `margin` to add space between your elements. When thinking of the box model, the space between elements is not necessarily part of the `content`.

Reply:
Thanks! Learning best practices is always appreciated! It's these invisible things that sooner or later make editing easier that I know make a difference in polish that results in hiring decisions.